CHANGELOG.md

```diff
@@ -22,6 +22,12 @@ Please document your changes in this format:
 
 ## [Unreleased]
 
+## [9.4.1] - 2021-07-21
```

Comment:
The release version and date is only added when we actually do a release, for now your changes will be "Unreleased", so they can stay in that section, see https://github.com/yunity/karrot-frontend/blob/master/CHANGELOG.md#unreleased for how that looks for merged PRs.

Reply:
Perfect, no need for dates